frog/imports/client/Wiki/index.js

```diff
@@ -687,6 +681,41 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
     );
   };
 
+  createPageModal = () =>
```

Comment:
Would it possible for you to separate your work on PageSettings to another PR since this PR is focused on fixing the import bugs?

Reply:
This is a part of refactor of modal